lib/telemetry.js

```diff
@@ -60,5 +67,31 @@ export const startPingLoop = () => {
     point.tag('arch', arch())
     point.tag('deployment_type', DEPLOYMENT_TYPE)
     writeClient.writePoint(point)
-  }, 10 * 60 * 1000)
+    writeClient.flush().catch(handleFlushError)
+    await timers.setTimeout(10 * 60 * 1000) // 10 minutes
+  }
+}
+
+export const runMachinesLoop = async () => {
+  while (true) {
+    const point = new Point('machine')
+    point.stringField('process_uuid', processUUID)
+    point.intField('cpu_count', cpus().length)
+    point.intField('cpu_speed_mhz', cpus()[0].speed)
+    const model = cpus()[0].model.toLowerCase()
+    point.tag(
+      'cpu_brand',
+      model.includes('intel')
+        ? 'intel'
+        : model.includes('amd')
+          ? 'amd'
+          : model.includes('apple')
+            ? 'apple'
+            : 'unknown'
+    )
+    point.intField('memory_total_b', totalmem())
+    writeClientMachines.writePoint(point)
+    writeClientMachines.flush().catch(handleFlushError)
+    await timers.setTimeout(86_400 * 1000) // 1 day
```

Comment:
Nitpick, feel free to ignore.

```suggestion
    await timers.setTimeout(86_400_000) // 1 day
```

The ping loop uses multiplication instead:

```js
await timers.setTimeout(10 * 60 * 1000) // 10 minutes
```

Maybe we should do the same in this place for consistency.

```suggestion
    await timers.setTimeout(24 * 3600 * 1000) // 1 day
```